lib/util/noop.js

```diff
@@ -1,6 +1,6 @@
 "use strict";
 
-var noop = function() {};
+var noop = Function.prototype;
```

Comment:
I tested in Firefox nightly and `Function.prototype` was 10x faster than `function() {}`